src/ddocs/search.rst

```diff
@@ -814,7 +814,7 @@ Drilldown
 You can restrict results to documents with a dimension equal to the specified label.
 Restrict the results by adding ``drilldown=["dimension","label"]`` to a search query. You
 can include multiple ``drilldown`` parameters to restrict results along multiple
-dimensions.
+dimensions: ``drilldown=[["dimension1","label1"], ["dimension2","label2"]]``
```

Comment:
can you show an example of multiple parameters (like the sentence before indicates is allowed) and then clarify that this new example allows you to do this with a single drilldown parameter?

Reply:
Done in 5c02705 . Let me know what you think.